app/src/main/res/values/strings.xml

```diff
@@ -3,45 +3,45 @@
     <string name="app_name" translatable="false">Seedvault</string>
     <string name="backup">Backup</string>
     <string name="data_management_label">Seedvault Backup</string>
-    <string name="current_destination_string">Backup status and settings</string>
+    <string name="current_destination_string">Back up status and settings</string>
 
     <string name="restore_backup_button">Restore backup</string>
 
     <!-- Settings -->
     <string name="settings_category_apps">App backup</string>
-    <string name="settings_backup">Backup my apps</string>
+    <string name="settings_backup">Back up my apps</string>
     <string name="settings_backup_location">Backup location</string>
     <string name="settings_backup_location_none">None</string>
     <string name="settings_backup_location_internal">Internal storage</string>
     <string name="settings_backup_last_backup_never">Never</string>
-    <string name="settings_info">All backups are encrypted on your phone. To restore from backup you will need your 12-word recovery code.</string>
-    <string name="settings_auto_restore_title">Automatic restore</string>
+    <string name="settings_info">All backups are encrypted on your device. Restore backups using your 12-word recovery code.</string>
+    <string name="settings_auto_restore_title">Auto-restore</string>
     <string name="settings_auto_restore_summary">When reinstalling an app, restore backed up settings and data.</string>
     <string name="settings_auto_restore_summary_usb">Note: Your %1$s needs to be plugged in for this to work.</string>
-    <string name="settings_category_app_data_backup">App data backup</string>
+    <string name="settings_category_app_data_backup">App-data backup</string>
     <string name="settings_backup_apk_title">App backup</string>
-    <string name="settings_backup_apk_summary">Back up the apps themselves. Otherwise, only app data would get backed up.</string>
-    <string name="settings_backup_apk_dialog_title">Really disable app backup?</string>
```

Comment:
The reason to not use "enable/disable" is it offers only higher entropy over "turn on/off".
A legitimate use would be if something can be on, while also disabled at some lower level, when this is a meaningful distinction to communicate to the user. I am not seeing it.

Reply:
"Turn on/off" would make sense here.